pkg/minikube/cni/custom.go

```diff
@@ -63,3 +69,20 @@ func (c Custom) Apply(r Runner) error {
 func (c Custom) CIDR() string {
 	return DefaultPodCIDR
 }
+
+func (c Custom) createFakeCNIConfig(r Runner) error {
```

Comment:
Can we rename this with a bit less stutter? This is in the `cni` package, after all. Perhaps:

`createFakeConfig` or `addFakeConfig`

Also, Since the idea of a fake cni config is a little weird, do you mind adding a single-line comment describing why we are doing this?

`// addFakeConfig adds a fake CNI configuration file to the node to (keep the kubelet from crashing?)`

Reply:
done